src/EnergyPlus/ThermalComfort.cc

```diff
@@ -243,6 +243,7 @@ namespace ThermalComfort {
     Array1D<Real64> ZoneOccHrs;
     bool useEpwData(false);
     Array1D<Real64> DailyAveOutTemp(30, 0.0);
+    Real64 avgDryBulbCEN(0.0);
```

Comment:
being moved here for the sake of unit tests

Reply:
Adding another global here doesn't seem necessary.  Can you just modify the function so that it either returns a Real64 or pass in a Real64 by reference and check the value after calling the function that way?

Reply:
I agree with your opinion about the globals, it is simply that I have failed so far to come up a more proper way to do the unit tests. Let me have some more thinking on this. Also if you have any other ideas, do let me know.